ui/layouts/OwnerApp.tsx

```diff
@@ -24,6 +26,12 @@ export const OwnerApp: React.FC<Props> = ({ children, title }) => {
   const { data: companies } = useCompanies()
   const { data: owner } = useOwner()
 
+  const [hint, setHint] = useState(localStorage.getItem('hintclosed') !== '1')
```

Comment:
Please use `React.useState` instead of importing `useState` – we use the `React.*` style throughout the app 🙂

Reply:
Also, since the initial state `localStorage.getItem...` is a blocking operation, I would suggest wrapping it inside a function. This way, it only gets executed inside the first render, and won't cause unnecessary blocking localstorage-reads on rerenders.

```suggestion
  const [hint, setHint] = React.useState(() => {
    return localStorage.getItem('hintclosed') !== '1')
  })
```